File: app/src/main/java/com/tistory/black_jin0427/myandroidarchitecture/api/model/Picture.java
package com.tistory.black_jin0427.myandroidarchitecture.api.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Picture implements Serializable {

    @SerializedName("large") public String large;

    @SerializedName("medium") public String medium;

    @SerializedName("thumbnail") public String thumbnail;
}

File: app/src/main/java/com/tistory/black_jin0427/myandroidarchitecture/api/model/Picture.kt
package com.tistory.black_jin0427.myandroidarchitecture.api.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

data class Picture (

    @SerializedName("large")
    var large: String?,

    @SerializedName("medium")
    var medium: String?,

    @SerializedName("thumbnail")
    var thumbnail: String?

) : Serializable
